app/controllers/sessions_controller.rb

```diff
@@ -2,6 +2,11 @@ class SessionsController < ApplicationController
   skip_before_action :request_login
 
   def new
+    if Settings.features.start_page
+      render "pages/start"
+      return
```

Comment:
```suggestion
      render "pages/start" and return
```

https://guides.rubyonrails.org/layouts_and_rendering.html#avoiding-double-render-errors

dangling `return`

Reply:
Rubocop wouldn't allow `and return` and we need to `return` so 🤷

Reply:
ain't picking a fight with Rubocop